README.md

```diff
@@ -91,25 +91,26 @@ To run the next command, install [Lychee](https://github.com/lycheeverse/lychee?
 
 If you use a different operating systems and a different methods of installation, the above command might require additional amendments. 
 
-The above command can also be run using the `docfx-util.ps1` Powershell script. Specifically, run `./docfx-util.ps1 -l <path/to/lychee.exe>` in the repo's root directory. 
+The above command can also be run using the `docfx-utils.ps1` Powershell script. Specifically, run `./docfx-utils.ps1 -l <path/to/lychee.exe>` in the repo's root directory. 
 
-All three link-checking commands can be run with the following command: `./docfx-util.ps1 -a` in the repo's root directory. This command additionally cleans remaining artifacts from past builds before performing all the link-checking commands. This is the most robust and expedient way to confirm that the repo will pass the link checks when pushed. 
+All three link-checking commands can be run with the following command: `./docfx-utils.ps1 -a` in the repo's root directory. This command additionally cleans remaining artifacts from past builds before performing all the link-checking commands. This is the most robust and expedient way to confirm that the repo will pass the link checks when pushed. 
 
 ### docfx-utils.ps1
 
 This is summary of docfx-utils.ps1 list members. They are described above, but they are also described below for ease-of-finding:
 
 - `docfx-utils.ps1 -c` cleans cached files/removes artifacts from previous builds.
 - `docfx-utils.ps1 -b` exports SVGs and builds the docs.
-- `docfx-utils.ps1 -l <path/to/lychee.exe>` checks for broken/missing links and references.
+- `docfx-utils.ps1 -d` check the markdown files for broken references using DocLinkChecker.
+- `docfx-utils.ps1 -l <path/to/lychee.exe>` checks for broken/missing links and references using lychee.
 - `docfx-utils.ps1 -a <path/to/lychee.exe>` performs all of the above steps.
 
 `docfx-utils.ps1 -l` and `docfx-utils.ps1 -a` will not run without a path to a lychee executable.   
 
 > [!NOTE]  
 > The docfx-utils.ps1 script (.ps1 PowerShell scripts in general) must be run in PowerShell.
 
-These commands do not serve the docs. Serve them by running `dotnet docfx serve`.
+These commands do not serve the docs. Serve them by running `dotnet docfx --serve`.
```

Comment:
I was curious why I needed to make this change, so I went back to the `man` page for this.

Here is the flag that I changed it to:
```
-s, --serve                        Host the generated documentation to a website
```

Meanwhile, this is the command that was originally there:
```
USAGE:
    docfx serve [directory] [OPTIONS]

ARGUMENTS:
    [directory]    Path to the directory to serve
```

While both of these could work, it does not work as-is, since the command requires the directory as an additional input. If you run it without the directory, this is the landing page that is generated by the command on my machine:

![image](https://github.com/user-attachments/assets/1072b104-5150-474f-8a88-36b09db04bab)

Meanwhile, the limitation for the flag is that it does rebuild the website before deploying it, which can take a few seconds to complete.

Therefore, the best thing to do would actually be to go back to the original edit, but then add `_site` as the directory input, which I tested and correctly serves the built documentation immediately. This of course assumes that the website was built prior to serving, but I would hope that is obvious. Either way, I can add a comment about it.